dataland-qa-service/src/main/kotlin/org/dataland/datalandqaservice/converters/DataTypeEnumConverter.kt

```diff
@@ -0,0 +1,15 @@
+package org.dataland.datalandqaservice.converters
+
+import jakarta.persistence.AttributeConverter
+import jakarta.persistence.Converter
+import org.dataland.datalandbackend.openApiClient.model.DataTypeEnum
+
+/**
+ * Converts DataTypeEnum entries in DatasetQaReviewLogEntity to database values and vice versa
+ */
+@Converter(autoApply = true)
+class DataTypeEnumConverter : AttributeConverter<DataTypeEnum, String> {
```

Comment:
True. It seems to me that using explicit types, in particular with the Converters, effectivly avoiding any potential issues due to saving EnumTypes in the database, may be good practice. I didn't change the QAReportEntity since I didn't want to touch anything Report related but stick to the scope of this ticket.

Reply:
Grundsätzlich ja, aber den dataTypeEnum werden wir zurückbauen. Der wird perspektivisch ersetzt durch Informationen die der specification service bereitstellt. Also in der Datenbank lieber mit Strings agieren. Dann ist das auch in Zukunft noch kompatibel.